app/controllers/hubstats/deploys_controller.rb

```diff
@@ -21,16 +21,20 @@ def index
 
     # show basic stats and pull requests from a single deploy
     def show
-      @deploy = Hubstats::Deploy.includes(:repo).includes(:pull_requests).find(params[:id])
+      @deploy = Hubstats::Deploy.includes(:repo, :pull_requests).find(params[:id])
       repo = @deploy.repo
       @pull_requests = @deploy.pull_requests
       pull_request_count = @pull_requests.length
       net_additions = find_net_additions(@deploy.id)
       comment_count = find_comment_count(@deploy.id)
+      additions = find_additions(@deploy.id)
+      deletions = find_deletions(@deploy.id)
       @stats_basics = {
```

Comment:
Defining local variables for all these values is a bit verbose.  Since you don't use the local variables for anything else in this method, consider reducing the verbosity here (andy memory usage for extra variables) and making this more succinct.

Also, you should pass the Deploy instance itself, instead of passing the id and going back to the database again.
```
@stats_basics = {
  pull_count: @pull_requests.length,
  net_additions: find_net_additions(@deploy),
  comment_count: find_comment_count(@deploy),
  additions: find_additions(@deploy),
  deletions: find_deletions(@deploy)
}
```